Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -279,6 +265,11 @@ - (void)setupToolbar {
 }
 
 - (void)updateToolbarItemsIfNeeded {
+    
+    if (!self.saveButtonController) {
```

Comment:
when don't we keep this as a lazy property? if it's set manually, we can't declare it as `nonnull`, which makes reasoning about the `saveButtonController` availability and configuration much simpler, IMO